primary.tf

```diff
@@ -1,9 +1,17 @@
+resource "random_string" "postfix" {
+  # There is a bug with unmanaged instance groups, adding a random string to the end
+  # of the primary instance name is a work around.
+  length  = "4"
+  special = false
+  upper   = false
+}
+
 resource "google_compute_instance" "primary" {
   # The number of primaries must be hard coded to 3 when Internal Production Mode
-  # is selected. Currently, that mode does not support scaling. In other modes, the
-  count = "${var.install_type == "ipm" ? 3 : var.primary_count}"
+  # is selected. Currently, that mode does not support scaling.
+  count = "${var.postgresql_user == "none" ? 3 : var.primary_count}"
```

Comment:
We're hardcoding this to 3 in AWS, probably do so here too.

Reply:
Updated this to be hardcoded to 3.